### PR TITLE
Use dataplane-operator in edpm_play

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

**Comment by raukadah** [comment]
Hello @fao89, I am testing this PR here: https://review.rdoproject.org/r/c/testproject/+/46993 and I am seeing two pods related to external compute
1. [dataplane-deployment-configure-network](https://logserver.rdoproject.org/93/46993/18/check/edpm-ansible-github-rdo-integration-centos-8-crc-singlenode-centos-9-external-compute/4b446f8/logs/controller/pod/dataplane-deployment-configure-networkmwhqb-7jfrk-logs.txt)

Here network related vars are not used in the above pod
```
edpm_network_config_template: _EDPM_NETWORK_CONFIG_TEMPLATE_
```
2. [deploy-external-dataplane-compute](https://logserver.rdoproject.org/93/46993/18/check/edpm-ansible-github-rdo-integration-centos-8-crc-singlenode-centos-9-external-compute/4b446f8/logs/controller/pod/deploy-external-dataplane-compute-p47jw-logs.txt)
Here only tripleo-repos tasks are executed.

Here are the logs of all pods: https://review.rdoproject.org/r/c/testproject/+/46993 
Since all these works are still in progress, so above execution might be expected like that.

**Comment by fao89** [comment]
I was planning to remove AEE entirely, but I needed to keep it for enabling tripleo repos,
I will rename to make it more meaningful

File: devsetup/edpm/edpm-play.yaml [hunk 1]
```diff
@@ -218,3 +54,172 @@ spec:
...
+      edpm_nodes_validation_validate_gateway_icmp: false
+
+      # 99-standalone-vars
+      ctlplane_dns_nameservers:
+      - 192.168.122.1
+      dns_search_domains: []
+      edpm_nova_compute_DEFAULT_transport_url:
+        "rabbit://guest:ar3TXOAYMaznW29fRLj2ILXKs@standalone.ctlplane.\
+        localdomain:5672/?ssl=0"
+      edpm_nova_compute_cache_memcache_servers: standalone.ctlplane.localdomain:11211
```
Comment (SeanMooney) [inline devsetup/edpm/edpm-play.yaml:145]: please remove all the 
edpm_nova_compute section as non of this is used and is explicitly intended to not be supported as a configuration interface by design.

configuration of nova is done via the CustomServiceConfig filed on the
NovaExternalComputeSpec cr that is generated as a drict passthrough fo

the CustomServiceConfig field on the  NovaExternalCompute Template on the OpenStackDataplaneNode or openstackDataplaneRole cr.

https://github.com/openstack-k8s-operators/dataplane-operator/pull/72/files#r1129176504

File: devsetup/edpm/edpm-play.yaml [hunk 2]
```diff
@@ -218,3 +54,91 @@ spec:
...
+
+      # 99-standalone-vars
+      ctlplane_dns_nameservers:
+      - 192.168.122.1
+      dns_search_domains: []
+      edpm_ovn_dbs:
+      - 192.168.24.1
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+  deployStrategy:
+    deploy: true
```
Comment (slagle) [inline devsetup/edpm/edpm-play.yaml:144]: can we just use the samples that I'm updating/adding in https://github.com/openstack-k8s-operators/dataplane-operator/pull/77 instead?

Comment (fao89) [reply devsetup/edpm/edpm-play.yaml:144]: yes, I'll update it

**Comment by fao89** [comment]
this PR may not be needed if https://github.com/openstack-k8s-operators/openstack-operator/pull/230 merges first

**Comment by fao89** [comment]
I started a new approach that uses the dataplane sample from the dataplane-operator repo: https://github.com/openstack-k8s-operators/install_yamls/pull/156

**Pull request closed without merging**